ui/core/sim.ts

```diff
@@ -271,8 +279,13 @@ export class Sim {
 		this.bulkSimStartEmitter.emit(TypedEvent.nextEventID(), request);
 
 		try {
-			const result = await this.workerPool.bulkSimAsync(request, onProgress);
+			const signals = this.signalManager.registerRunning(request.requestId, RequestTypes.BulkSim);
+			const result = await this.workerPool.bulkSimAsync(request, onProgress, signals);
 			if (result.errorResult != '') {
+				if (result.errorResult.includes('aborted')) {
+					// TODO: Abort feedback?
```

Comment:
Could perhaps add a toast if you want to show feedback

Reply:
I changed error_result into a ErrorOutcome object to disambiguate the abort outcome. Added a toast if it was an abort.

Reply:
Nice!